src/style.css

```diff
@@ -0,0 +1,22 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+ul[role='list'] {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+html:focus-within {
+  scroll-behavior: smooth;
+}
+
+body {
+  margin: 0;
+  min-height: 100vh;
+  text-rendering: optimizeSpeed;
+  line-height: 1.5;
+}
```

Comment:
- [ ] You are good to go but it would be better if you align the add score section with recent scores horizontally as in the wireframe 
- Yours 
![Screen Shot 2022-10-11 at 15 36 48](https://user-images.githubusercontent.com/61790587/195121646-75ef9711-72bf-4846-a9c6-97938a3192a9.png)
- Wireframe 
![Screen Shot 2022-10-11 at 15 37 16](https://user-images.githubusercontent.com/61790587/195121674-d0080539-f086-4663-ade5-51dc84008f11.png)